crates/nu-command/src/filters/upsert.rs

```diff
@@ -66,15 +66,15 @@ impl Command for Upsert {
             result: Some(Value::List { vals: vec![Value::Record { cols: vec!["project".into(), "authors".into()], vals: vec![Value::test_string("nu"), Value::test_string("Andrés,JT,Yehuda")], span: Span::test_data()}], span: Span::test_data()}),
         },
         Example {
-            description: "Upsert a value int a list, updating an existing value",
+            description: "Upsert an int into a list, updating an existing value",
             example: "[1 2 3] | upsert 0 2",
             result: Some(Value::List {
                 vals: vec![Value::test_int(2), Value::test_int(2), Value::test_int(3)],
                 span: Span::test_data(),
             }),
         },
         Example {
-            description: "Upsert a value int a list, inserting a new value",
+            description: "Upsert an int into a list, inserting a new value",
```

Comment:
```suggestion
            description: "Upsert an int into a list, inserting a new value based on the index",
```

Maybe worth clarifying that for lists it operates on indices instead of the value identities

Reply:
I changed both of the to say `based on the index`. thanks!

Reply:
Thanks!